src/riak_cs_riak_client.erl

```diff
@@ -331,6 +331,8 @@ get_user_with_pbc(MasterPbc, Key, true) ->
 get_user_with_pbc(MasterPbc, Key, false) ->
     case strong_get_user_with_pbc(MasterPbc, Key) of
         {ok, _} = OK -> OK;
+        {error, <<"{pr_val_unsatisfied,", _/binary>>} ->
```

Comment:
> when n_val is not 3. 

Exactly :bulb: 

> could 3,1 happen in faulty cluster and that's the case we want to warn operators?

Worth consideration :thought_balloon: Just my opinion, cluster membership should be monitored in riak side, not from riak cs logs. The reason is monitoring target should be one-time event (node-up, node-down) but riak cs can, at best, provide noisy, repeating-over-and-over log lines of warnings.